app/models/ocelot/stanzas/InstructionStanza.scala

```diff
@@ -27,14 +27,19 @@ case class InstructionStanza(text: Int, override val next: Seq[String], link: Op
 
 object InstructionStanza {
 
-  implicit val instructionReads: Reads[InstructionStanza] = {
-
+  implicit val reads: Reads[InstructionStanza] =
     ((JsPath \ "text").read[Int] and
       (JsPath \ "next").read[Seq[String]](minLength[Seq[String]](1)) and
       (JsPath \ "link").readNullable[Int] and
       (JsPath \ "stack").read[Boolean])(InstructionStanza.apply _)
 
-  }
+  implicit val owrites: OWrites[InstructionStanza] =
```

Comment:
Hi Chris, can you explain the reason for using OWrites please? I guess it may have something to do with accessing the values using the keys properties.

Reply:
See above